source/expectations/data_values/ut_compound_data_value.tpb

```diff
@@ -218,6 +218,28 @@ create or replace type body ut_compound_data_value as
       return l_column;
     end;
     
+    procedure produce_pk_hash(a_pk_hash_extr_string varchar2, a_join_by_xpath in varchar2, a_diff_id raw) is
+    begin
+      execute immediate 'merge into ' || l_ut_owner || '.ut_compound_data_diff_tmp tgt
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a WHEN OTHERS exception handler to ensure that all exceptions are trapped. ([plsql:ExecuteImmediateTrapExceptionsCheck](https://sonarcloud.io/organizations/utplsql/rules#rule_key=plsql%3AExecuteImmediateTrapExceptionsCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=utPLSQL&pullRequest=760&issues=AWY_N4wenzfF1D4JsH2F&open=AWY_N4wenzfF1D4JsH2F)

[ISSUE_KEY]: AWY_N4wenzfF1D4JsH2F